src/ui/assets-and-images.md

```diff
@@ -162,14 +112,26 @@ Future<String> loadAsset() async {
 
 ### Loading images
 
-Flutter can load resolution-appropriate images for
-the current device pixel ratio.
+To load an image, use the [`AssetImage`][]
+class in a widget's `build()` method.
 
-#### Declaring resolution-aware image assets {#resolution-aware}
+For example, your app can load the background
+image from the asset declarations above:
 
-[`AssetImage`][] understands how to map a logical requested
+<?code-excerpt "main.dart (BackgroundImage)"?>
+```dart
+return const Image(image: AssetImage('graphics/background.png'));
+```
+
+### Resolution-aware image assets {#resolution-aware}
+
+Flutter can load resolution-appropriate images for
+the current [device pixel ratio][].
+
+Specifically, [`AssetImage`][] understands how to map a logical requested
 asset onto one that most closely matches the current
 [device pixel ratio][].
```

Comment:
**nit:** Remove the adverb. It adds nothing to the understanding. Not sure if `AssetImage` can "understand", but that's a minor point.
```suggestion
[`AssetImage`][] understands how to map a logical requested
asset onto one that most closely matches the current
[device pixel ratio][].
```

Reply:
Addressed with commit [remove anthropomorphism of class and adverb](https://github.com/flutter/website/pull/8710/commits/754dc833c7eb3a5a5ad1133a49bc47732436eb73)